node/src/lib.rs

```diff
@@ -126,3 +126,235 @@ pub(crate) fn new_rng() -> NodeRng {
 pub(crate) fn new_rng() -> NodeRng {
     NodeRng::new()
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
Seems like this should maybe just live inside the sync leap test module, since that's the only place these helpers are consumed?

If you feel that they will be used more widely in the future, then we have the `crate::testing` module which might be more appropriate?

Reply:
Yes, good idea. Historically, this code was moved around since initially it was used to test the [batch block header fetch](https://github.com/casper-network/casper-node/commit/ee3057a41094d86cbc654463d1fe6ee45ce61aae) in the original fast sync implementation.

Addressed in 2d9e61d1ab95cdf8d85bfbf2f13afe56b632a9bf